README.md

```diff
@@ -51,7 +51,7 @@ Refer to pytest's documentation or tutorials to understand how to start tests or
 
 Examples:
 ```
-pytest tests/storage/ext/test_ext_sr.py --hosts=10.0.0.1
+pytest tests/storage/ext/test_ext_sr.py --hosts=10.0.0.1 --sr-disk=auto
```

Comment:
Since --sr-disk=auto might cause unwanted data losses, maybe:

```suggestion
pytest tests/storage/ext/test_ext_sr.py --hosts=10.0.0.1 --sr-disk=[device (e.g "sdb" or "auto"]
```